common/djangoapps/pipeline_mako/tests/test_render.py

```diff
@@ -0,0 +1,33 @@
+""" Tests for rendering functions in the mako pipeline. """
+
+from django.test import TestCase
+from pipeline_mako import render_require_js_path_overrides
+
+
+class RequireJSPathOverridesTest(TestCase):
+    """Test RequireJS path overrides. """
+
+    OVERRIDES = [
+        'js/vendor/jquery.min.js',
+        'js/vendor/backbone-min.js',
+        'js/vendor/text.js'
+    ]
+
+    OVERRIDES_JS = [
+        "<script type=\"text/javascript\">",
+        "(function (require) {",
+        "require.config({",
+        "paths: {",
+        "'js/vendor/jquery.min': 'js/vendor/jquery.min',",
```

Comment:
I think it would because it would verify that your code uses those settings/methods to determine what to output. With the test as it is, it would pass with code that simply does: 

````
new_paths.append("'{module}': '{module}'".format(module=url_path.replace('.js', ''))